frontend/src/components/Trigger.tsx

```diff
@@ -52,6 +52,7 @@ interface TriggerProps {
     maxConcurrentRuns?: string;
     catchup: boolean;
   }) => void;
+  isMaxConcurrentRunValid: boolean;
```

Comment:
Since the error message is written within <Trigger> component, it can validate the input itself, so there is no need to pass `isMaxConcurrentRunValid` as a prop.